### PR TITLE
Refactor Releases docs

### PR DESCRIPTION
**Pull request by dfarrell07**
* Clean up `code style` usages.
* Use **bold** to highlight key ideas, not terms/projects.
* Fix various capitalization issues.
* Consistent punctuation, like ending sentences with periods.
* Fix version number of second release.
* Link to PRs instead of using ambiguous number.

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

File: src/content/releases/_index.en.md [hunk 1]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
+* HostNetwork to remote Service connectivity fixes for AWS clusters ([submariner#736](https://github.com/submariner-io/submariner/issues/736)).
 * The project's codebase quality and readability has been improved using various linters.
 
 ## v0.5.0 Lighthouse service discovery alignment
 
 > This release mainly focused on continuing the alignment of Lighthouse's service discovery support with the [Kubernetes Multicluster
 > Services KEP][MCS KEP].
 
-* **Lighthouse** has been modified per the [Kubernetes Multicluster Services KEP][MCS KEP] as follows:
+* **Lighthouse** has been modified per the **[Kubernetes Multicluster Services KEP][MCS KEP]** as follows:
```
Comment (nyechiel) [inline src/content/releases/_index.en.md:39]:
```suggestion
* Lighthouse has been modified per the **[Kubernetes Multicluster Services KEP][MCS KEP]** as follows:
```

File: src/content/releases/_index.en.md [hunk 2]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
+* Support for updated strongSwan ([submariner#288](https://github.com/submariner-io/submariner/issues/288)).
+* Better iptables detection for some hosts ([submariner#227](https://github.com/submariner-io/submariner/pull/227)).
+
+> `subctl` and the Submariner Operator have the following improvements:
+
+* Support for `verify-connectivity` checks between two connected clusters.
+* Deployment of Submariner gateways based on `DaemonSet` instead of `Deployment`.
+* Rename `submariner` Pods to `submariner-gateway` Pods for clarity.
+* Print version details on crash (`subctl`).
+* Stop storing IPSEC key on Broker during `deploy-broker`, now it's only contained into the `.subm` file.
```
Comment (nyechiel) [inline src/content/releases/_index.en.md:139]:
```suggestion
* Stop storing IPsec key on Broker during `deploy-broker`, now it's only contained into the `.subm` file.
```

File: src/content/releases/_index.en.md [hunk 3]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
+* Version command for `subctl`.
+* Nicer spinners during deployment (thanks to kind).
 
 ## v0.0.3 -- KubeCon NA 2019
 
-Submariner has been greatly enhanced to allow Operators to deploy into Kubernetes clusters without the necessity for layer-2 adjacency for
+Submariner has been greatly enhanced to allow Operators to deploy into Kubernetes clusters without the necessity for Layer 2 adjacency for
```
Comment (nyechiel) [inline src/content/releases/_index.en.md:145]:
```suggestion
Submariner has been greatly enhanced to allow operators to deploy into Kubernetes clusters without the necessity for Layer 2 adjacency for
```

Comment (nyechiel) [reply src/content/releases/_index.en.md:145]: (here we are referring to the Kubernetes operators/adminstrators and not OLM)

Comment (dfarrell07) [reply src/content/releases/_index.en.md:145]: Maybe "administrators" would be more clear, given that we also built an Operator in that same time period?

Comment (nyechiel) [reply src/content/releases/_index.en.md]: @dfarrell07 yeah, administrators would work.

**Review by nyechiel — APPROVED** [review]

File: src/content/releases/_index.en.md [hunk 4]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
+  Broker cluster. Therefore, the new `subctl` cannot be used on brownfield Globalnet deployments where this information
+  was stored as part of `broker-info.subm`.
+* `subctl` now supports **joining multiple clusters in parallel** without having to explicitly specify the `globalnet-cidr` for the
   cluster to work around this issue. The `globalnet-cidr` will automatically be allocated by `subctl` for each cluster.
 * The separate `--operator-image` parameter has been removed from `subctl join` and the `--repository` and `--version`
   parameters are now used for all images.
 * The Submariner Operator status now includes `Gateway` information.
-* Closed technical requirements for Submariner to become a CNFC project, including _Developer Certificate of Origin_ compliance,
-  and source code linting.
+* Closed technical requirements for Submariner to become a **CNFC project**, including _Developer Certificate of Origin_ compliance
```
Comment (skitt) [inline src/content/releases/_index.en.md:55]:
```suggestion
* Closed technical requirements for Submariner to become a **CNCF project**, including _Developer Certificate of Origin_ compliance
```

File: src/content/releases/_index.en.md [hunk 5]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
+  * Be aware of a Service's backend Pods so as not to announce IP addresses for Services that have no active Pods.
+  * Use Round Robin IP resolution for Services available in multiple clusters.
+  * Enable service discovery by default for `subctl` deployments.
+* `subctl` auto-detects the cluster ID from the `kubeconfig` file's information when possible.
+* Submariner's Pods now shutdown gracefully and do proper cleanup which **reduces downtime** during Gateway failover.
+* The Operator now automatically exports **Prometheus metrics**; these integrate seamlessly with OpenShift Prometheus if user
   workload monitoring is enabled, and can be included in any other Prometheus setup.
 * Minimum Kubernetes version is now 1.17.
-* HostNetwork to remote Service connectivity fixes for AWS clusters ([Issue 736](https://github.com/submariner-io/submariner/issues/736)).
+* HostNetwork to remote Service connectivity fixes for AWS clusters ([submariner#736](https://github.com/submariner-io/submariner/issues/736)).
```
Comment (skitt) [inline src/content/releases/_index.en.md:31]:
```suggestion
* HostNetwork to remote Service connectivity [fixes for AWS clusters](https://github.com/submariner-io/submariner/issues/736).
```

File: src/content/releases/_index.en.md [hunk 6]
```diff
@@ -11,137 +11,141 @@ weight = 30
...
-> This release is focused on removing the KubeFed dependency from Lighthouse, improving the user experience
-> and adding experimental WireGuard support as an alternative to IPsec
+> This release is focused on removing the KubeFed dependency from Lighthouse, improving the user experience,
+> and adding experimental WireGuard support as an alternative to IPsec.
 
-* Lighthouse no longer depends KubeFed. All metadata exchange is handled over the Broker as MultiClusterService CRs.
-* Experimental **Wireguard** support has been added as a pluggable CableDriver option in addition to the current default IPsec.
-* Submariner reports the active and passive gateways as a gateway.submariner.io resource.
+* Lighthouse **no longer depends on KubeFed**. All metadata exchange is handled over the Broker as `MultiClusterService` CRs.
+* Experimental **Wireguard** support has been added as a pluggable `CableDriver` option in addition to the current default IPsec.
```
Comment (skitt) [inline src/content/releases/_index.en.md:83]:
```suggestion
* Experimental **WireGuard** support has been added as a pluggable `CableDriver` option in addition to the current default IPsec.
```

**Review by skitt — CHANGES_REQUESTED** [review]

**Comment by dfarrell07** [comment]
Great fixes @skitt, thanks!

**Review by tpantelis — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**